### PR TITLE
adjustments re requires.auth

### PR DESCRIPTION
**Pull request by sjvans**

### CODE REVIEW

**Review by chgeo — APPROVED** [review]

**Review by danjoa — APPROVED** [review]
👍

File: package.json [hunk 1]
```diff
@@ -35,12 +35,9 @@
       "db": {
         "kind": "sql"
       },
-      "uaa": {
-        "kind": "xsuaa"
-      },
       "auth": {
         "[development]": {
-          "strategy": "dummy"
+          "kind": "dummy-auth"
```
Comment (johannes-vogel) [inline package.json:40]: Isn't that a breaking change for our customers if they now need to specify the kind instead of the strategy?

Comment (chgeo) [reply package.json:40]: No, we don't want to be breaking, i.e. the old `strategy:dummy` still has to work in the runtime.  Please check if this is the case.

This is about showcasing the _recommended_ approach, which is why we change it.

Comment (johannes-vogel) [reply package.json:40]: got it

Comment (sjvans) [reply package.json:40]: > No, we don't want to be breaking, i.e. the old `strategy:dummy` still has to work in the runtime. Please check if this is the case.
> 
> This is about showcasing the _recommended_ approach, which is why we change it.

Still works (with depreciation warning)

**Comment by stewsk** [comment]
has become obsolete

**Pull request closed without merging**